libraries/bxdf/usd_preview_surface.mtlx

```diff
@@ -12,7 +12,7 @@
     <input name="useSpecularWorkflow" type="integer" value="0" />
     <input name="specularColor" type="color3" value="0, 0, 0" />
     <input name="metallic" type="float" value="0" />
-    <input name="roughness" type="float" value="0.01" />
+    <input name="roughness" type="float" value="0.5" />
```

Comment:
Hi Jerry,
This clearly looks incorrect as it's defaulting to the clearcoat roughness value.

@nicolassavva-autodesk , @ashwinbhat : This value is correct as of v2.2 definition, and we've marked this definition as v2.3 already. I'm guessing there will be some shader translation differences that will arise now and need to be compensated for ? 

Think we should at least know if there are any issues which may arise first before we check this in?
Thanks.

Reply:
Yes, we should change the mtlx to the prescribed defaults so as to match the spec.

I believe we always explicitly specify the "roughness" input value in our internal conversion workflows to usd_preview_surface so this might not be an issue as far as we are concerned (though it is still a good idea to batch some additional tests).

Reply:
Thanks. Let's assume this is the general case.